var/www/html/tools/MemcachedAdmin/Config/Memcache.php

```diff
@@ -0,0 +1,33 @@
+<?php
+return array (
```

Comment:
This file gets replaced with the output of consul-template, yes?

Perhaps it should be removed or remain with just a line explaining how/where it's generated?